<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackFaceAddDlg</class>
 <widget class="QDialog" name="BlackFaceAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Face</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widToolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblScene">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblFaceDetection">
     <property name="text">
      <string>Face Detection</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="NoKeyGraphicsView" name="viewDetection">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>102</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>102</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDown">
       <property name="text">
        <string>↓ Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUp">
       <property name="text">
        <string>↑ Up</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEnrolledFace">
     <property name="text">
      <string>Enrolled Face</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="NoKeyGraphicsView" name="viewEnroll">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>102</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>102</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddImage">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/image.png</normaloff>:/images/image.png</iconset>
   </property>
   <property name="text">
    <string>Select Image</string>
   </property>
   <property name="toolTip">
    <string>Select Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoKeyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>nokeygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainResource.qrc"/>
 </resources>
 <connections/>
</ui>
